.taskmaster/agent-terminals.py

```
@@ -0,0 +1,694 @@
+#!/usr/bin/env python3
+"""
+Individual Agent Terminal Interface
+Provides Claude Code-like control over each agent with real-time output streaming
+"""
+
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)